### PR TITLE
Handle null animation effect

### PR DESCRIPTION
**Pull request by steveblock**

### CODE REVIEW

File: test/testcases/unit-test-null-effect.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+test(function() {
+  assert_equals(new Animation(target, null, 1.0).effect, null);
+}, "Animation.effect should be null if effect is null");
+
+test(function() {
+  assert_equals(new Animation(target, null, 1.0).clone().effect, null);
+}, "Cloning an animation with a null effect should yield a null effect");
+
+test(function() { document.timeline.play(new Animation(target, null, 1.0)); },
+    "Playing an animation with a nulleffect does not cause an exception");
```
Comment (dstoc) [inline test/testcases/unit-test-null-effect.html]: nulleffect -> null effect

**Comment by dstoc** [comment]
lgtm

**Comment by mithro** [comment]
lgtm.

**Comment by steveblock** [comment]
Thanks - fixed 'nulleffect'

**Pull request merged**